CoreFoundation/String.subproj/CFString.h

```diff
@@ -175,20 +177,14 @@ struct __CFConstStr {
 #endif // defined(__LP64__) || defined(__LLP64__)
 };
```

Comment:
Should the structure definition be updated to match the reality of the compiler?

Reply:
This structure definition is used only when the compiler is not involved, and is now ifdef'd away. I checked that the actual struct used by CFStringRef (`__CFString`) does use intptr_t lengths as we discussed.

Reply:
Any reason to not make the two be identical?